composer.json

```diff
@@ -26,7 +26,7 @@
 	],
 	"require": {
 		"ext-curl": "*",
-		"php": "8.0.*||8.1.*||8.2.*",
+		"php": "8.1.* || 8.2.*",
```

Comment:
Why is php 8.3 omitted hear ? (already before)

I would suggest to use

```
  "php": "^8.1 || ^8.2"
```

here. In the wild, such a contraint is used as

* support php 8.1 and 8.2
* allow php 8.3 for testing/execution without offical support and without using unsecure `--ignore-platfrom-req`

Not sure about the policy at all - from a personal point of view all php version allowed by supported core version should be allowed/supported by a extension - which would mean php7.4 for a v11 support. Using newer php language feature should be ommited.

As this is already broken, not sure about the min version raise - eventually leave out
the "readonly" for the CPP and keep php8.0 (CPP is possible since 8.0, readonly since 8.1). Not sure about overall version policy of this extension.